index.js

```diff
@@ -48,149 +49,139 @@ Redis.prototype.connect = function () {
   });
 };
 
-Redis.prototype.handle_client_oper_action = function (action, key, callback) {
-  callback = does_callback_exist(callback);
-
-  this.client[action](key, (err, data) => {
-    handle_err_log(err);
-    callback(err, data);
-  });
+Redis.prototype.disconnect = async function () {
+  return this.client.quit();
 };
 
-Redis.prototype.get = function (key, callback) {
-  this.handle_client_oper_action('get', key, callback);
+Redis.prototype.handle_client_oper_action = async function (action, key) {
+  return tryExecuteAndLogError(this.client[action](key));
```

Comment:
minor: I prefer snake_case instead of camelCase

Reply:
applied